common/constants.js

```diff
@@ -282,7 +282,11 @@ module.exports = Object.freeze({
     ATTRIBUTES: 'attributes',
     PLANS: 'plans'
   },
-
+  ENCRYPTION: {
+    AES_256_ALGORITHM: 'aes-256-ctr',
```

Comment:
Instead of using **aes-256-ctr** can we use **aes-256-gcm** as GCM achieves **confidentiality** and **integrity** both where CTR supports only confidentiality?

Reply:
might be good to check if there are some corp/org guidelines on the same.

Reply:
change of plan!.....keeping it generic so that the consumer can choose the algorithm!

Reply:
Dont think its a good idea to parameterise as it leads to additional validation at EncryptionManager on the algorithm name passed. Also different modules could encrypt in different algorithms. Its best we enforce this for two reasons:
1. Standardisation across the product
2. Also, avoid errors which could occur when modules might change the algorithm across releases.

So if we want to fix it to `aes-256-ctr` lets go ahead and fix it in this PR and when we really want to start using the encryption then we can check for org guidelines.

Reply:
Agree @jagadish-kb . 
I have done the changes accordingly.